cbsh.nuspec

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter
+enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+    <metadata>
+        <id>cbsh.portable</id>
+        <version>$version$</version>
+        <owners>Charles Dixon, Jack Westwood</owners>
+        <title>Couchbase Shell (Portable)</title>
+        <authors>Charles Dixon, Jack Westwood</authors>
+        <projectUrl>https://couchbase.sh</projectUrl>
+        <projectSourceUrl>https://github.com/couchbaselabs/couchbase-shell</projectSourceUrl>
+        <bugTrackerUrl>https://github.com/couchbaselabs/couchbase-shell/issues</bugTrackerUrl>
+        <tags>cli portable couchbase</tags>
+        <description>The modern and fun shell for Couchbase server and Cloud</description>
```

Comment:
This should probably say "Couchbase Server and Couchbase Capella", since we've updated the branding.

Reply:
Done